BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollector.java

```diff
@@ -117,4 +117,32 @@ private String getPayPalClientMetadataId(Context context, Configuration configur
         }
         return "";
     }
+
+    /**
+     * Collects device information for fraud identification purposes from PayPal only.
+     *
+     * @param context           Android Context
+     * @param riskCorrelationId Correlation id to associate with data collection
+     * @param callback          {@link DataCollectorCallback}
+     */
```

Comment:
On iOS we have this note for this function since we recommend using `collectDeviceData`:
```
This returns a raw client metadata ID, which is not the correct format for device data when creating a transaction. Instead, it is recommended to use `collectDeviceData`.
```
Worth calling that out here too?

Reply:
Definitely yeah. I'll add this in.